go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/lyft/flyteplugins/go/tasks/pluginmachinery/utils"
 	"k8s.io/client-go/kubernetes/scheme"
 
-	sparkOp "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1beta1"
+	sparkOp "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1beta2"
```

Comment:
thats it no changes in the API?

Reply:
There are some change like types of some fields in the Spec has changed but we don't use that in flyteplugins i.e. only set those config via spark_conf. The  one other change is see is that status is now a sub-resource and the operator needs additional perms for the same.